.travis/before_script.sh

```diff
@@ -14,3 +14,20 @@ sudo chown -R travis:travis /var/lib/pulp
 sudo cp .travis/server.yaml /etc/pulp/server.yaml
 
 echo "SECRET_KEY: \"$(cat /dev/urandom | tr -dc 'a-z0-9!@#$%^&*(\-_=+)' | head -c 50)\"" | sudo tee -a /etc/pulp/server.yaml
+
+# Run migrations.
+export DJANGO_SETTINGS_MODULE=pulpcore.app.settings
```

Comment:
I think all of this should be moved underneath the if statement.  We don't need migrations or workers for a docs test.  I think? that in it's original location, it was gated by an if statement that would just exit earlier in the script if it was a docs test.

Reply:
The 'make html' command downloads the OpenAPI schema from a running pulp instance on localhost. This requires a running web server.